docs/serving/services/using-queue-extensions.md

```diff
@@ -0,0 +1,80 @@
+# Using Queue Proxy Extensions
+
+Once your cluster is setup to allow services to use Queue Proxy Extensions, a service can decide which extensions it wish to use and how to configure such extensions. Activating and configuring extensions is described here.
+
+## Prerequisites for preparing the cluster
+
+1. Make sure you are using a Queue Proxy Image that was built with the extensions that you wish to use - See [Queue Proxy Extensions](../queue-extensions.md).
+1. Make sure that the cluster config-features is set with: `queueproxy.mount-podinfo: allowed`  - See [Enabling Queue Proxy Pod Info](../configuration/feature-flags.md#queue-proxy-pod-info) for more details
+
+## Overview
+
+A service can activate and configure queue proxy extensions by adding `qpextension.knative.dev/*` annotations under the: `spec.template.metadata` of the service custom resource definition.
```

Comment:
re queue proxy extensions use capitalization and naming consistently throughout docs please